<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>679</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>110</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Log73 by EU2FM (text format)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AppResources.qrc">
    <normaloff>:/images/73.png</normaloff>:/images/73.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoBulletList</set>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>81</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CALLSIGN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>81</y>
      <width>69</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tx No</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>81</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rx No</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>83</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Region</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>120</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>120</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>681</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Табло</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>770</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string>Табло</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <column>
     <property name="text">
      <string>tur</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CALLSIGN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TxN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RxN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Region</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>add items</string>
    </property>
   </widget>
   <widget class="QLabel" name="clock">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>38</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 38pt &quot;Ubuntu&quot;;
background-color: rgb(0, 0, 0);
color: rgb(0, 241, 0);</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otmenaButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="otmenaButton_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Записать</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>761</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>tableWidget</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>clock</zorder>
   <zorder>textEdit</zorder>
   <zorder>otmenaButton</zorder>
   <zorder>otmenaButton_2</zorder>
   <zorder>tableView_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSetup"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionShowMap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuMap"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/images/Save.png</normaloff>:/images/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/images/Saveas.png</normaloff>:/images/Saveas.png</iconset>
   </property>
   <property name="text">
    <string>Export to txt</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionShowMap">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/images/map.PNG</normaloff>:/images/map.PNG</iconset>
   </property>
   <property name="text">
    <string>Belarus</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Content</string>
   </property>
  </action>
  <action name="actionEdit_text">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/images/Edit.png</normaloff>:/images/Edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit text</string>
   </property>
  </action>
  <action name="actionEdit_done">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/images/EditDone.png</normaloff>:/images/EditDone.png</iconset>
   </property>
   <property name="text">
    <string>Edit done</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
  <action name="actionBelarus">
   <property name="text">
    <string>Belarus</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>otmenaButton_2</tabstop>
  <tabstop>otmenaButton</tabstop>
 </tabstops>
 <resources>
  <include location="AppResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>isQSOforTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>retPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>retPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>retPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_4</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>retPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otmenaButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otmenaButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>retPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>retPressed()</slot>
  <slot>MySlotTimer()</slot>
  <slot>isQSOforTab()</slot>
 </slots>
</ui>
